data_handling/controllers/articles.controllers.js

```diff
@@ -64,3 +64,20 @@ exports.postArticleComment = (req, res, next) => {
       });
   };
 
+  exports.patchArticleVotes = (req, res, next) => {
+    const voteInc = req.body.inc_votes;
+    const id = req.params.article_id;
```

Comment:
You could destructure these directly from the req.body and req.params for example: 
**const {id} = req.params**